packaging/standalone/src/main/distribution/shell-scripts/bin/neo4j-shared.sh

```diff
@@ -126,8 +124,8 @@ _setup_calculated_paths() {
   if [[ -z "${NEO4J_HOME:-}" ]]; then
     NEO4J_HOME="$(cd "$(dirname "$0")"/.. && pwd)"
   fi
-  : "${NEO4J_CONF:=conf}"
-  readonly NEO4J_ROOT NEO4J_CONF
+  : "${NEO4J_CONF:=$(resolve_path conf)}"
```

Comment:
There is no need for `resolve_path` here: `: "${NEO4J_CONF:="${NEO4J_HOME}/conf"}"`.